sdk/spring/azure-spring-boot/pom.xml

```diff
@@ -197,6 +197,12 @@
       <version>2.11.2</version> <!-- {x-version-update;com.fasterxml.jackson.core:jackson-databind;external_dependency} -->
     </dependency>
 
+    <dependency>
```

Comment:
Dependencies should be introduced after discussing with our architect. https://azure.github.io/azure-sdk/java_implementation.html#java-dependencies-archboard

@JonathanGiles Is this dependency okay with you?

Reply:
@JonathanGiles @conniey I removed this dependency in b864a8eb603ab5630f0af894acaf17ec44f789bc by using a `String` instead of an `Optional<String>` for `odataNextLink` in [UserGroups.java](https://github.com/Azure/azure-sdk-for-java/pull/14305/files#diff-83972daa53535c303b68814b85159b59R16)